config/routes.rb

```diff
@@ -1,11 +1,14 @@
 Rails.application.routes.draw do
+  # Disable PUT for now since rails sends these :update and they aren't really the same thing.
+  def put(*args, &block); end
```

Comment:
Minor, but per rubocop style, these should be `*_args, &_block`, or even better, you don't even need to capture the block so...

```ruby
def put(*); end
```